tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -265,13 +265,180 @@ Ramdump received successfully..!
 DumpParser Script provides two interfaces: CUI and GUI
 
 ### DumpParser using CUI
+
+#### To get call stack using RAM dump
 1. Run Ramdump Parser Script
 ```
 cd $TIZENRT_BASEDIR/tools/dump_tool/
 python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
 2. See the Output
 
+#### To display Debug Symbols/Crash point using assert logs
+1. Copy crash logs
+    First copy the crash logs to a file named log_file in tools/dump_tool/`<log_file>`
+2. Run Script in tools/dump_tool path
+    $ python3 ramdumpParser.py -e `<Elf path>` -t `<Log file path>`
+
+    ex)
+    $ python3 ramdumpParser.py -e ../../build/output/bin/app2_dbg -t ./log_file
+
+Example Call Stack Output for App crash is as follows:
+```
+*************************************************************
+dump_file         : None
+log_file          : .log_file
+elf_file  : ../../../output/app2_dbg
+*************************************************************
+
+Number of applicaions : 2
+App[1] is : app1
+App[2] is : app2
+
+----------------------------------------------------------
+App Crash point is as follows:
+[Crash point corresponding to the R14 register value (LR)]
+App name : app2
+0x000014d6
+main
+/root/tizenrt/loadable_apps/loadable_sample/wifiapp/wifiapp.c:121
+
+[Probable crash point corresponding to the R15 register value (PC) might be -4 or -8 bytes]
+App name : app2
+0x000014e5
+main
+/root/tizenrt/loadable_apps/loadable_sample/wifiapp/wifiapp.c:113
```

Comment:
1. I think non-familiar on ARM outweigh familiar on ARM. (I mean many peoples don't know ARM register itself)
How about writing the title like below?
```md
[ Caller - return address (LR) - of the function which makes the crash ]
```
2. I don't know what you intend at `(PC) might be -4 or -8 bytes`. Does user need to check -4 or -8 from the printed address? or Is it already checked in the printed?
3. Could you write the item name?
```md
app name : xx
code addr? symbol addr? : xx
function name : xx
file : xxx
```

Reply:
As you know, my English is poor. When you take my suggestion, please re-write the sentence.

Reply:
1) [ Caller - return address (LR) - of the function which has caused the crash ] sounds ok ?
    Should I add this information as and when a match is found for PC & LR values after the existing log like below?
    [ Crash point corresponding to the R14 register value (LR) ]
    [ Caller - return address (LR) - of the function which has caused the crash ] 
2) User needs to check -4 or -8 from the printed address.
3) I will modify

Reply:
> Should I add this information as and when a match is found for PC & LR values after the existing log like below?
[ Crash point corresponding to the R14 register value (LR) ]
[ Caller - return address (LR) - of the function which has caused the crash ]

Sorry I don't get your point.